scripts/publish/src/getPackages.js

```diff
@@ -0,0 +1,7 @@
+const { exec } = require('shelljs');
+
+// Get all packages in lerna scope.
+module.exports = () =>
+  exec('npx lerna ls --parseable', { silent: true })
```

Comment:
Is there a reason why you're using lerna scope instead of looking at `/packages/*/*`? We're not using lerna publish, so it might be better to just simplify things and just look in the directory structure.

Reply:
I chose this as it would always match the scope of lerna, which we use to build and install packages. But yes I could just use `fs`